core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -136,6 +137,14 @@ public String toString() {
 
   protected abstract String defaultWarehouseLocation(TableIdentifier tableIdentifier);
 
+  protected String formatTableName(TableIdentifier table, boolean uniqueTableLocation) {
```

Comment:
maybe this should go into `LocationUtil` to better indicate that this is about "locations". At the very least we should find a better name, since `format` is rather misleading

Reply:
I moved it to LocationUtil and renamed to `getTableNameComponent`